packages/blade/src/components/Dropdown/DropdownHeaderFooter.tsx

```diff
@@ -0,0 +1,127 @@
+/* eslint-disable @typescript-eslint/prefer-nullish-coalescing */
+import React from 'react';
+import { BaseFooter } from '../BaseHeaderFooter/BaseFooter';
+import { BaseHeader } from '../BaseHeaderFooter/BaseHeader';
+import type { BaseHeaderProps } from '../BaseHeaderFooter/BaseHeader';
+import type { BaseFooterProps } from '../BaseHeaderFooter/BaseFooter';
+import { useDropdown } from './useDropdown';
+import BaseBox from '~components/Box/BaseBox';
+import { assignWithoutSideEffects, isReactNative, makeAccessible, MetaConstants } from '~utils';
+import { useBottomSheetAndDropdownGlue } from '~components/BottomSheet/BottomSheetContext';
+
+type DropdownHeaderProps = Pick<
+  BaseHeaderProps,
+  'title' | 'subtitle' | 'leading' | 'trailing' | 'titleSuffix' | 'testID'
+>;
+
+const _DropdownHeader = ({
+  title,
+  subtitle,
+  leading,
+  titleSuffix,
+  trailing,
+  testID,
+}: DropdownHeaderProps): React.ReactElement => {
+  return (
+    <BaseBox
+      overflow={'auto' as never}
```

Comment:
why as never?

Reply:
to go around that `PickCSSByPlatform` in common files